index.html

```diff
@@ -694,9 +694,12 @@ <h2><dfn>readable</dfn> attribute</h2>
         <ol>
           <li>
             Let |desiredSize| be the
-            <a href="https://streams.spec.whatwg.org/#desired-size-to-fill-a-streams-internal-queue">desired size</a>
-            of [=this=].{{[[readable]]}}'s
-            <a href="https://streams.spec.whatwg.org/#internal-queues">internal queue</a>.
+            [=ReadableStream/desired size to fill up to the high water mark=] for
+            [=this=].{{[[readable]]}}.
```

Comment:
@domenic, sorry, could you change all the internal slots to be namespaced? This depends on #145. 

```suggestion
            [=this=].{{SerialPort/[[readable]]}}.
```